config/initializers/rails_admin.rb

```diff
@@ -67,7 +67,7 @@ def edit_user_link
     field :abbreviated_titles, :serialized do
       html_attributes rows: '6', cols: '70'
     end
-    fields :canonical_title, :synopsis, :slug, :subtype, :poster_image, :cover_image,
+    fields :canonical_title, :description, :slug, :subtype, :poster_image, :cover_image,
```

Comment:
This will probably break Rails Admin, it needs to be set up as `:serialized` like `titles` is

Reply:
ahhh yeah, you are right.